src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomUap.java

```diff
@@ -76,17 +76,17 @@ public void adsFandomUapOnTopicMobile(String pageType, String pageName, long atf
 
   private void verifyUapAtf(long atfId, String slotName, AdsFandomObject fandomPage) {
     fandomPage.verifySlot(AdsFandomContent.TOP_LEADERBOARD);
-    Assertion.assertEquals(atfId, fandomPage.getLineItemId(AdsFandomContent.TOP_LEADERBOARD));
+    Assertion.assertEquals(fandomPage.getLineItemId(AdsFandomContent.TOP_LEADERBOARD), atfId);
 
     fandomPage.verifySlot(slotName);
-    Assertion.assertEquals(atfId, fandomPage.getLineItemId(AdsFandomContent.TOP_BOXAD));
+    Assertion.assertEquals(fandomPage.getLineItemId(AdsFandomContent.TOP_BOXAD), atfId);
   }
 
   private void verifyUapBtf(long btfId, String slotName, String bottomSlotName, AdsFandomObject fandomPage) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyUapBtf" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)